mlflow/protos/service.proto

```diff
@@ -55,6 +55,17 @@ service MlflowService {
     };
   }
 
+  rpc getExperimentByName (GetExperimentByName) returns (GetExperimentByName.Response) {
```

Comment:
Could you add a brief comment header like:
```
// Gets the experiment with the specified name.
```
above this line? The comment will show up in autogenerated REST API docs / make it easier to use the API.